scripts/send-message-from-millau-rialto.sh

```diff
@@ -11,9 +11,9 @@ case "$1" in
 		RUST_LOG=runtime=trace,substrate-relay=trace,bridge=trace \
 		./target/debug/substrate-relay send-message millau-to-rialto \
 			--millau-host localhost \
-			--millau-port 20044 \
-			--millau-signer //Dave \
-			--rialto-signer //Dave \
+			--millau-port 9944 \
```

Comment:
So this changes the use case a little bit, and we should make a note of that in the documentation.

The ports originally used here match our Docker deployment. That meant you could use this script for testing a message which this after spinning up a network using `docker-compose`.

This changes that, and it's fine as long as we also mention what the requirements for running a full message sync network are in the documentation. That's to say: you need nodes + header relays + message relays running _before_ this can even work.

We also do want to keep the docs related to Docker since it's pretty useful for some quick testing.

Reply:
i see the local scripts as an alternative version of docker compose in case the user does not want to use docker or whatever. We could say in the documentation that this is the alternative way of the docker compose but with same ( or similar connection details) as port numbers for example.

Reply:
Let's use an environment variable in both scripts and default to either docker (my preference) or local setup.
Then in case you want to override you could simply
```
export MILLAU_PORT=9944
export RIALTO_PORT=9955
./local_scripts/...
```
or even create an `.env` file.

Reply:
> Let's use an environment variable in both scripts and default to either docker (my preference) or local setup.
> Then in case you want to override you could simply
> 
> ```
> export MILLAU_PORT=9944
> export RIALTO_PORT=9955
> ./local_scripts/...
> ```
> 
> or even create an `.env` file.

Yeah, something like this could work. Docker is also my preference (if you haven't already noticed) since getting to the point where you can even send a message requires _at minimum_ six different processes running.